examples/tutorials/colabs/ECCV_2020_Navigation.ipynb

```diff
@@ -140,7 +135,12 @@
     "else:\n",
     "    show_video = False\n",
     "    make_video = False\n",
-    "    display = False"
+    "    display = False\n",
+    "\n",
+    "# import the maps module alone for topdown mapping\n",
+    "if display:\n",
+    "    sys.path.insert(1, \"/content/habitat-api/\")\n",
```

Comment:
Why is this necessary? Habitat-API should already be in the PATH in the system wide install?

Reply:
The bug was solved by #731